### DISCUSSION TITLE
Docker CUDA Images only for 11.1.1 and 10.0.130

### DISCUSSION THREAD

**Discussion by tomekrut** [flashlight/flashlight#482] [Q&A] [answered]
Hi, Am I correct, that you only build cuda images for two cuda versions? I can't find anything with 10.2 or 11.0 just 11.1.1 and 10.0.130...
Thanks

**Comment by tlikhomanenko** [comment, marked as answer]
Yep, we don't build images for all possible cuda version. You simply can modify the dockerfiles and build necessary version of cuda docker.
